modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -2084,7 +2084,7 @@
                   "errors": [
                     {
                       "title": "Unprocessable Entity",
-                      "detail": "Veteran cannot file for type 'survivor'"
```

Comment:
interesting, why did these change for this PR? do we just need to update this branch?